standalone/src/test/java/io/pravega/local/PravegaEmulatorResource.java

```diff
@@ -71,6 +71,7 @@ public PravegaEmulatorResource(boolean authEnabled, boolean tlsEnabled, boolean
                 .enableRestServer(restEnabled)
                 .enableAuth(authEnabled)
                 .enableTls(tlsEnabled)
+                .tlsProtocolVersion(TestUtils.getTlsProtocolVersion())
```

Comment:
Please add tests to verify strict mode for standalone. This would be an automated way to verify different TLS protocol versions.

Reply:
Added tests to verify strict mode for standalone.